src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+
+Decentralized lending works without either party having to identify themselves. Instead the borrower must put up collateral that the lender will automatically receive if their loan is not repaid.
+
+Loans can be peer-to-peer, meaning a borrower will borrow directly from a specific lender. Or they can be pool-based where lenders provide liquidity to a pool that borrowers can borrow from.
+
+Some lenders even accept NFTs as collateral. NFTs are a deed to a unique asset, like a painting. [More on NFTs](/nfts/)
+
+A more experimental form of decentralized lending lets you borrow without collateral or providing any personal information...
+
+#### Flash loans {#flash-loans}
+
+Flash loans are not advised if you're not a technical expert but they hint at what might be possible to everyone in the future. With Ethereum you can take out huge loans without putting up any collateral whatsoever.
+
+It works on the premise that the loan is taken out and paid back within the same transaction. If it can't be paid back, the transaction reverts as if nothing ever happened.
+
+This means a lot of logic must be included in a very bespoke transaction. A simple example might be someone using a flash loan to borrow as much of an asset at one price so they can sell it on a different exchange where the price is higher.
+
+So in a single transaction the following happens:
+
+- You borrow X amount of $asset at $1.00 from exchange A
+- You sell X $asset on exchange B for $1.10
+- You pay back loan to exchange A
+- You keep the profit minus the transaction fee
+
+If exchange B's supply dropped suddenly and the user wasn't able to buy enoughh to cover the original loan, the transaction would simply fail.
+
+### Start saving with crypto {#saving}
+
+You can earn interest on your crypto by lending it. For example, you can lend your Dai to a product like Aave and receive Aave Dai (aDai) which is a token that represents your loaned Dai. Your aDai will increase based on the interest rates and you can see it grow in real-time in your wallet. Your aDai also lets you withdraw the equivalent in Dai at any time.
+
+Etheruem has also seen a new way to save emerge: no-loss lotteries.
+
+No-loss lotteries like PoolTogether are a fun and innovative new way to save money. Buy tickets for a lottery that you can't lose. Only one winner is chosen every week, but your tickets roll over and you can withdraw at any time. The prize pot is generated by the interest gained on all the tickets.
+
+### Swapping tokens {#swaps}
+
+There are lots of tokens on Ethereum. Decentralized exchanges let you trade your tokens while always having control of your assets.
+
+When you use a centralized exchange you have to deposit your assets before the trade and trust them to look after them. While your assets are deposited, they're at risk as centralized exchanges are attractive targets for hackers.
```

Comment:
I'd start this section with this paragraph on centralized exchanges.

Traditionally to trade you'd likely need to rely on a centralized exchange... + the rest of what you have.